service/gcs/runtime/runc/runc_test.go

```diff
@@ -182,7 +174,7 @@ var _ = Describe("runC", func() {
 			c  runtime.Container
 		)
 		JustBeforeEach(func() {
-			c, err = rtime.CreateContainer(id, bundle, createAllStdioOptions)
+			c, err = rtime.CreateContainer(id, bundle, &stdio.ConnectionSet{})
```

Comment:
We should probably pass connections into this so that the code which manages writing to them etc. is actually run.